src/components/InputFields.jsx

```diff
@@ -23,6 +24,7 @@ const InputFields = ({
         handleDateOfBirthChange(selectedDate);
     };
 
+    
```

Comment:
These `calculated` functions are used directly in the render function and not as a callback to any action. For that reason, it is better to calculate these immediately and memoize them to avoid recalculating them at every re-render (same issue with the current implementation as well).

read: https://react.dev/reference/react/useMemo